README.md

```diff
@@ -91,19 +91,38 @@ To run you will use the "shares" command
 
 **Input parameters:**
 - keystore (ks) = The path to either a validator keystore file or a folder that contains multiple validator keystore files. If a folder is provided, it will split in bulk all the keystore files within it according to the additional arguments provided
-- password (ps) = The keystore file encryption password, if a folder was provided the password will be used for all keystore files in the folder
+- password (ps) = The keystore file encryption password. For one keystore, enter the absolute path. For multiple keystores, enter only the password file name.
 - operator-ids (oids) = Comma-separated list of operator IDs. The amount must be 3f+1 compatible
 - operator-keys (oks) = Comma-separated list of operator keys (same sequence as operator ids). The amount must be 3f+1 compatible
 - output-folder (of) = Target folder path to output the key shares file
 - owner-address (oa) = The cluster owner address (in the SSV contract)
 - owner-nonce (on) = The validator registration nonce of the account (owner address) within the SSV contract (increments after each validator registration), obtained using the ssv-scanner tool
 
 ```bash
+
+6 directories, 20 files
 # single file
-yarn cli shares --keystore=keystore.json --password=test --operator-ids=1,2,3,4 --operator-keys=LS..,LS..,LS..,LS.. --output-folder=./ --owner-address=... --owner-nonce=..
+yarn cli shares --keystore=./keystore.json --password=./password.txt --operator-ids=1,2,3,4 --operator-keys=LS..,LS..,LS..,LS.. --output-folder=./ --owner-address=... --owner-nonce=..
 
 # folder with multiple keystore files
-yarn cli shares --keystore=./keystore-files --password=test --operator-ids=1,2,3,4 --operator-keys=LS..,LS..,LS..,LS.. --output-folder=./ --owner-address=... --owner-nonce=..
+## folder structure
```

Comment:
Separate this usecase with this sub header
Add comment to underline that _--password_ parameter is actual password file name
"folder structure for keystore files with different password"

Add sub header for one password and multiple files. Command should be the same as before change

Reply:
Updated.